Desktop/mainwindow.cpp

```diff
@@ -575,6 +575,22 @@ void MainWindow::setQmlImportPaths()
 	Log::log() << std::endl;
 }
 
+void MainWindow::generateWrappersForModule(QString moduleName)
+{
+	static QString wrapperAlreadyGeneratedForThisModule; // The moduleInstallationSucceeded is called several times: this prevents to create several times the analyses.
+
+	if (wrapperAlreadyGeneratedForThisModule == moduleName) return;
+	wrapperAlreadyGeneratedForThisModule = moduleName;
+
+	for (auto analysisEntry : _dynamicModules->dynamicModule(moduleName)->menu())
+		if (analysisEntry->isAnalysis())
+		{
+			Analysis* analysis = _analyses->create(analysisEntry);
+			analysis->setWatchFileChange(false);
```

Comment:
`setWatchFileChange` which file and why?
I saw the code and I still dont know... Is this for the QML or what?
Also if the default is `true` then every Analysis is watching "file" but why would every normal analysis started need to watch a file at all?